src/pages/block/Block.tsx

```diff
@@ -199,50 +182,37 @@ const Block: React.FC<Props> = (props: Props) => {
               <div className="detail-tile script-tile">
                 <div className="script-label-and-copy-row">
                   <label>INVOCATION SCRIPT</label>{' '}
-                  {chain === 'neo2' ? (
-                    <Copy text={block ? block.script?.invocation : ''} />
-                  ) : (
-                    <Copy
-                      text={
-                        block && block.witnesses
-                          ? block.witnesses[0].invocation
-                          : ''
-                      }
-                    />
-                  )}
+                  <Copy
+                    text={
+                      block && block.witnesses
+                        ? block.witnesses[0].invocation
+                        : ''
+                    }
+                  />
                 </div>
                 <span>
-                  {chain === 'neo2'
-                    ? !isLoading && block && block.script?.invocation
-                    : !isLoading &&
-                      block &&
-                      block.witnesses &&
-                      block.witnesses[0].invocation}
+                  {!isLoading &&
+                    block &&
+                    block.witnesses &&
+                    block.witnesses[0].invocation}
                 </span>
               </div>
               <div className="detail-tile script-tile">
                 <div className="script-label-and-copy-row">
                   <label>VERIFICATION SCRIPT</label>
-
-                  {chain === 'neo2' ? (
-                    <Copy text={block ? block.script?.verification : ''} />
-                  ) : (
-                    <Copy
-                      text={
-                        block && block.witnesses
-                          ? block.witnesses[0].verification
-                          : ''
-                      }
-                    />
-                  )}
+                  <Copy
+                    text={
+                      block && block.witnesses
+                        ? block.witnesses[0].verification
+                        : ''
+                    }
```

Comment:
This expression should just be:

```javascript
block?.witnesses?[0]?.verification || ''
```

Reply:
same as previous, my IDE complains. Note that I didn't create this expression. I just removed the Neo2 'branch' of it and the rest already existed.